nohup: appending output to 'nohup.out'
[WARN] [1627552131.478221]: Inbound TCP/IP connection failed: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.
[WARN] [1627552112.092986]: Inbound TCP/IP connection failed: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.
[WARN] [1627552352.880401]: Inbound TCP/IP connection failed: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.
[WARN] [1627552352.344214]: Inbound TCP/IP connection failed: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.
[WARN] [1627552220.951377]: Inbound TCP/IP connection failed: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.
[WARN] [1627552353.525622]: Inbound TCP/IP connection failed: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.
/home/ubuntu/ws/src/seaowl/asv_referee/nodes/asv_referee_node2.py:409: RuntimeWarning: divide by zero encountered in true_divide
  return np.minimum(1,1/x)
[WARN] [1627552443.156809]: Inbound TCP/IP connection failed: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.
[31m================================================================================REQUIRED process [reaper_node-2] has died!
process has finished cleanly
log file: /home/ubuntu/.ros/log/fc9d27d2-f051-11eb-8cff-63bfa3b0c376/reaper_node-2*.log
Initiating shutdown!
================================================================================[0m
[31m================================================================================REQUIRED process [reaper_node-34] has died!
process has finished cleanly
log file: /home/ubuntu/.ros/log/fc9d27d2-f051-11eb-8cff-63bfa3b0c376/reaper_node-34*.log
Initiating shutdown!
================================================================================[0m
Traceback (most recent call last):
  File "executable9.py", line 151, in <module>
    run(serial, params, uuid)
  File "executable9.py", line 109, in run
    launch.start()
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/parent.py", line 305, in start
    self._start_infrastructure()
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/parent.py", line 263, in _start_infrastructure
    self._start_server()
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/parent.py", line 208, in _start_server
    raise RLException("pm is not initialized")
roslaunch.core.RLException: pm is not initialized
